mne_connectivity/vector_ar/var.py

```diff
@@ -199,7 +201,12 @@ def vector_auto_regression(
         X, Y = _construct_var_eqns(data, lags=lags, l2_reg=l2_reg)
 
         if cv_alphas is not None:
-            reg = RidgeCV(alphas=cv_alphas, cv=5).fit(X, Y)
+            with warnings.catch_warnings():
+                warnings.filterwarnings(
+                    action='ignore',
+                    message="Ill-conditioned matrix"
+                )
```

Comment:
Why do we need this?

Reply:
RidgeCV tests out an array of alpha values and some of them do not regularize the matrix enough to avoid an ill-conditioned matrix error. If the user sees many of these messages pop up, they may think that something is going wrong, when in fact the expected behavior of the function is happening. RidgeCV will choose the best alpha value and that will be from an instance when this error was not thrown.